easybuild/easyconfigs/p/PSORTb/PSORTb-3.0.4-foss-2016a-Perl-5.22.1.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PerlModule'
+
+name = 'PSORTb'
+version = '3.0.4'
+
+homepage = 'http://psort.org/psortb/index.html'
+description = """PSORTb v3.0.4 is the most precise bacterial localization prediction tool available."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://www.psort.org/download/']
+sources = ['bio-tools-psort-all.%(version)s.tar.gz']
+
+patches = ['PSORTb-%(version)s-libpsortb.patch']
+
+perl = 'Perl'
+perlver = '5.22.1'
+versionsuffix = '-%s-%s' % (perl, perlver)
+
+dependencies = [
+    (perl, perlver),
+    ('libpsortb','1.0'),
+    ('BioPerl','1.6.924','-Perl-5.22.1'),
+    ('BLAST+','2.3.0', '-Python-2.7.11'),
+    ('pftools','2.3'),
+]
+
+options = {'modulename': 'Bio::Tools::PSort'}
+
+preconfigopts = "export PSORTROOT=%(installdir)s && "
+
+postinstallcmds = [
+   "cp -r psort/* %(installdir)s/.",
+]
+
+perlmajver = perlver.split('.')[0]
```

Comment:
I think it's OK to hardcode the `5` below, since Perl6 is a very different beast